docs/src/tutorials/tutorial_6.md

```diff
@@ -0,0 +1,313 @@
+# **Tutorial 6:** Using Animations.jl to Create something with more Pep!
+
+In the last couple of tutorials you've learned the basics of Javis and some of the more advanced stuff like [`SubAction`](@ref).
+
+## Our goal
+
+The goal of this tutorial is to explain a new feature we have added in v0.2 of Javis. Before this every animation as basically linear.
+What I mean by this is: If you move an object from `A` to `B` using [`Translation`](@ref) it would do so in a linear and therefore boring fashion.
+
+We'll create an interesting loading animation for this. It consists of five circles which move from the center to the outside rotate around the center and back to the origin. During this process they appear and disappear as well as changing color.
```

Comment:
Maybe just say "In this tutorial, we'll create a loading animation that looks like this:" and show the final gif below it.

Reply:
I don't wanna show the final gif at the beginning as I think it destroys the joy a bit :joy: Do you think that's okay?

Reply:
Do you show what you create though in your tutorials? Or did I show it before in mine :stuck_out_tongue: Why not stick to what we have? Scrolling through it pretty easy/fast. Or am I wrong?

Reply:
Hmmm... Actually, we haven't been consistent with this. Instead, I think we should be fine with how it is. Especially given that we do state learning outcomes at the beginning.